### ISSUE TITLE
The linear evaluation is very low

### ISSUE THREAD

**Issue by yanghhx** [LinguoLi/CrosSCLR#3]
I have run your code and the linear evaluation accuracy is only 45.84%. When I set the "param.requires_grad =True" in line 44,49,54 of linear.py, the linear evalution accuracy of NTU60 view can reach 83.3%. However , the operation may be the fine-tuning operation that is described in your paper. I am very confusing.

**Comment by 2220666627** [comment]
I have the same issue, the linear evaluation is very low, only set the "param.requires_grad =True" of linear.py can reach the 83.3% accuracy.

**Comment by Xiangyue-Zhang** [comment]
same here!

**Comment by Xiangyue-Zhang** [comment]
Maybe try to check your torch version, make sure it is 1.4.0, works for me.
